certgen/certProcessor/src/main/java/org/incredible/CertsConstant.java

```diff
@@ -189,35 +184,49 @@ public String getPreview(String preview) {
         return Boolean.toString(false);
     }
 
-    public Map<String, Object> getStorageParamsFromEvn() {
+    public Map<String, Object> getStorageParamsFromEvn(String storageType) {
```

Comment:
An alternate to add parameters is to create another function explicit. Imagine this will always give a public container (say, for example) and the other function will give a private container.
This way existing callers wouldn't change and it becomes easy to figure out who uses private container.

Reply:
if will do seperate method for public and private then we have four method for aws and azure

Reply:
Number of methods don't cause any harm. It beautifies the code instead.